rskj-core/src/main/java/co/rsk/config/RemascConfig.java

```diff
@@ -45,6 +45,14 @@
 
     private long lateUncleInclusionPunishmentDivisor = 20;
 
+    // Multiplier and Divisor for paid fees comparison in selection rule
+    private long paidFeesMultiplier = 2;
+    private long paidFeesDivisor = 1;
+
+    public long getPaidFeesMultiplier() { return paidFeesMultiplier; }
+
+    public long getPaidFeesDivisor() { return paidFeesDivisor; }
```

Comment:
Divisor is always 1 ?

Reply:
Now, yes. It was discussed with Mr. Lerner